src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+
+    public Ball() {
```

Comment:
do we call the default constructor anywhere in the code?

Reply:
It seems at the beginning

Reply:
probably you meant that it was declared in the beginning of the class. 
When I say `called` I mean the place somewhere in the code when we use this constructor  and create a new instances.